pkg/handler/v2/routes_v2.go

```diff
@@ -7835,3 +7852,109 @@ func (r Routing) listOperatingSystemProfilesForCluster() http.Handler {
 		r.defaultServerOptions()...,
 	)
 }
+
+// swagger:route GET /api/v2/projects/{project_id}/clusters/{cluster_id}/serviceaccount project listClusterServiceAccount
+//
+//	List service accounts in cluster.
+//
+//	Produces:
+//	- application/json
+//
+//	Responses:
+//	  default: errorResponse
+//	  200: []ClusterServiceAccount
+//	  401: empty
+//	  403: empty
+func (r Routing) listClusterServiceAccount() http.Handler {
+	return httptransport.NewServer(
+		endpoint.Chain(
+			middleware.TokenVerifier(r.tokenVerifiers, r.userProvider),
+			middleware.UserSaver(r.userProvider),
+			middleware.SetClusterProvider(r.clusterProviderGetter, r.seedsGetter),
+			middleware.SetPrivilegedClusterProvider(r.clusterProviderGetter, r.seedsGetter),
+		)(cluster.ListClusterSAEndpoint(r.userInfoGetter, r.projectProvider, r.privilegedProjectProvider)),
+		cluster.DecodeListClusterServiceAccount,
+		handler.EncodeJSON,
+		r.defaultServerOptions()...,
+	)
+}
+
+// swagger:route POST /api/v2/projects/{project_id}/clusters/{cluster_id}/serviceaccount project createClusterServiceAccount
+//
+//	Creates a service account in cluster.
+//
+//	Consumes:
+//	- application/json
+//
+//	Produces:
+//	- application/json
+//
+//	Responses:
+//	  default: errorResponse
+//	  200: ClusterServiceAccount
```

Comment:
nit: it's a good practice to return 201 status after creating resources. Remember to update response encoder as well.
```suggestion
//	  201: ClusterServiceAccount
```

Reply:
done :)